postgres/master/drop_kpi_tables_in_kc.sh

```diff
@@ -0,0 +1,165 @@
+ constance_config
+ taggit_taggeditem
+ taggit_tag
+ kpi_objectpermission
+ kpi_assetsnapshot
+ kpi_assetfile
+ kpi_assetversion
+ kpi_asset
+ kpi_collection
+ kpi_importtask
+ kpi_authorizedapplication
+ kpi_taguid
+ kpi_onetimeauthenticationkey
+ kpi_usercollectionsubscription
+ kpi_exporttask
+ hub_sitewidemessage
+ hub_configurationfile
+ hub_formbuilderpreference
+ hub_extrauserdetail
+ hub_perusersetting
+ registration_registrationprofile
+ hook_hooklog
+ hook_hook
+ external_integrations_corsmodel
+ help_inappmessageuserinteractions
+ help_inappmessagefile
+ help_inappmessage
+)
+
+kpi_tables_single_quoted_csv=$(echo "${KPI_TABLES[@]}" | sed "s/^/'/;s/ /','/g;s/$/'/")
+actual_kpi_tables_count=$(
+    psql -Xqt -h localhost \
+        -U "$POSTGRES_USER" \
+        -d "$KC_POSTGRES_DB" \
+        -c "SELECT COUNT(*) FROM pg_tables WHERE tablename in ($kpi_tables_single_quoted_csv);"
+)
+if [ "$actual_kpi_tables_count" -ne ${#KPI_TABLES[@]} ]
+then
+    echo -n "The KoBoCAT database, \`$KC_POSTGRES_DB\`, does not contain the needed KPI tables. "
+    echo -n 'If this installation never used a single, shared database for both KPI and KoBoCAT, '
+    echo 'then this script is not needed. Otherwise, the database configuration may be incorrect.'
+    echo "For help, visit $HELP_PAGE."
+    exit 1
+fi
+
+matching_dbs_count=$(
+    psql -Xqt -h localhost \
+        -U "$POSTGRES_USER" \
+        -d postgres \
+        -c 'SELECT COUNT(*) FROM pg_database WHERE datname='"'$KPI_POSTGRES_DB';"
+)
+if [ "$matching_dbs_count" -eq 1 ]
+then
+    echo "Safety check: the KPI database \`$KPI_POSTGRES_DB\` exists (OK), "
+    asset_count=$(
+        psql -Xqt -h localhost \
+            -U "$POSTGRES_USER" \
+            -d "$KPI_POSTGRES_DB" \
+            -c 'SELECT COUNT(*) FROM kpi_asset;' \
+        | sed 's/ //g'
+    )
+    if [ $asset_count -eq 0 ]
+    then
+        echo "The KPI database \`$KPI_POSTGRES_DB\` appears to be empty because it contains no assets."
+        echo "This script cannot continue with an empty assets table in KPI database \`$KPI_POSTGRES_DB\`."
+        exit 1
+    fi
+else
+    echo -n "The KPI database \`$KPI_POSTGRES_DB\` does not exist, or "
+    matching_tables_count=$(
+        psql -Xqt -h localhost \
+            -U "$POSTGRES_USER" \
+            -d "$KPI_POSTGRES_DB" \
+            -c 'SELECT COUNT(*) FROM pg_tables WHERE tablename='"'kpi_asset';"
+    )
+    if [ "$matching_tables_count" -eq 0 ]
+    then
+        echo "it appears to be empty because it does not contain an assets table."
+        echo "This script cannot continue with an empty KPI database \`$KPI_POSTGRES_DB\`."
+        exit 1
+    fi
+fi
+
+kpi_last_actual_migration=$(
+    psql -Xqt -h localhost \
+        -U "$POSTGRES_USER" \
+        -d "$KPI_POSTGRES_DB" \
+        -c "SELECT name FROM django_migrations WHERE app='kpi' ORDER BY id DESC LIMIT 1;" \
+    | sed 's/ //g'
+)
+if [ "$kpi_last_actual_migration" = "$KPI_LAST_EXPECTED_MIGRATION" ]
```

Comment:
Maybe we should check the last KoBoCAT migration? Our main concern is that we don't delete any of KoBoCAT's tables